core/src/main/java/com/github/stephenvinouze/materialnumberpickercore/MaterialNumberPicker.kt

```diff
@@ -164,7 +156,6 @@ class MaterialNumberPicker : NumberPicker {
             paint.color = textColor
             paint.textSize = textSize.toFloat()
             paint.typeface = typeface
-
             val childEditText = (0 until childCount).map { getChildAt(it) as? EditText }.firstOrNull()
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/StephenVinouze/MaterialNumberPicker/blob/ae55545cc2574a05af41ab00efd15ced1c007f9e/core/src/main/java/com/github/stephenvinouze/materialnumberpickercore/MaterialNumberPicker.kt#L159"></span>Exceeded max line length (100) (cannot be auto-corrected)</td>
    </tr>
  </tbody>
</table>